doc/src/main/java/com/jagrosh/jdautilities/doc/ConvertedBy.java

```diff
@@ -19,13 +19,13 @@
 
 /**
  * Specifies an {@link java.lang.annotation.Annotation Annotation} can be converted
- * using the specified {@link com.jagrosh.jdautilities.doc.DocConverter DocConverter}
+ * using the specified {@link DocConverter DocConverter}
```

Comment:
A reminder to fix these documentation formatting inconsistencies

Reply:
...and this